scala3doc/src/dotty/dokka/tasty/SyntheticSupport.scala

```diff
@@ -66,7 +66,7 @@ trait SyntheticsSupport:
   def hackGetAllMembers(using QuoteContext)(rsym: qctx.reflect.Symbol): List[qctx.reflect.Symbol] = {
     import qctx.reflect._
     import dotty.tools.dotc
-    given ctx as dotc.core.Contexts.Context = rootContext.asInstanceOf
+    given ctx as dotc.core.Contexts.Context = qctx.asInstanceOf[scala.quoted.runtime.impl.QuoteContextImpl].ctx
```

Comment:
Not sure why it needs the cast --- ideally APIs in `scala.quote._` will be enough for usage, so that there is no need to use both reflect API and compiler API.

Reply:
This `hacky` method needs to be removed. Nobody should ever do this kind of cast.

Reply:
That is in the list of TODOs